[NewProduct] Fairphone

adriens opened the pull request:
# :grey_question: About

As I currently wanted to buy a new mobile phone, I wanted to benchmark them. 

:bulb: I saw that [Fairphone](https://www.fairphone.com/en/) was not listed... so here we go :smile_cat:  

# :ticket: Related issue

- https://github.com/endoflife-date/endoflife.date/issues/2338

adriens committed:
feat(fairphone) : Add Fairphone device

adriens committed:
Fairphone cleanup

cf https://github.com/endoflife-date/endoflife.date/pull/2339#issuecomment-1399663029

captn3m0 commented on products/fairphone.md:
```diff
@@ -0,0 +1,42 @@
+    releaseDate: 2013-12-01
+---
+
+> Fairphone is a Dutch electronics manufacturer that designs and produces smartphones with the goal of having a lower environmental footprint and better social impact than is common in the industry.
```
This should focus on the product, not the company.

captn3m0 replied:
How about something like:

> Fairphone is a line of smartphones that are designed with the goal of having
> a lower environmental footprint and better social impact than is common in the industry.

captn3m0 replied:
This is still pending.

captn3m0 commented on products/fairphone.md:
```diff
@@ -0,0 +1,42 @@
+
+> Fairphone is a Dutch electronics manufacturer that designs and produces smartphones with the goal of having a lower environmental footprint and better social impact than is common in the industry.
+
+> We aim to support Fairphone 4 with security and software upgrades for approximately five years from its launch
```
The tone should always be third-person neutral present tense (See Style Guide on the wiki). "we" here is the Fairphone company, and we can't write on their behalf, unless we're quoting, and we should avoid it there as well to avoid confusion.

Something like:

Fairphone 4 is supported with security and software upgrades for approximately 5 years from its launch.

would be better.

However, this will get covered by the table already, so probably better to skip this line.
```suggestion
```

captn3m0 commented on products/fairphone.md:
```diff
@@ -0,0 +1,42 @@
+> We aim to support Fairphone 4 with security and software upgrades for approximately five years from its launch
+> date (September 2021).
+> For Fairphone 4, we guarantee software support until the end of 2026, this includes an upgrade to Android 12 and, later on, Android 13.
+> We also aim to provide upgrades for Android 14 and 15. However, since Qualcomm will not offer support in the development, we cannot guarantee them.
```
```suggestion
Fairphone 4 is guaranteed an upgrade to Android 13. Updates to Android 14/15 [might happen on a best effort basis](https://support.fairphone.com/hc/en-us/articles/4405858006545-FP4-Fairphone-OS-Android-11-).
```

Avoid repeating information already in the table.

captn3m0 commented on products/fairphone.md:
```diff
@@ -0,0 +1,42 @@
+releasePolicyLink: https://forum.fairphone.com/t/fairphone-product-release-cycle/52652
+discontinuedColumn: true
+activeSupportColumn: false
+eolColumn: Supported
```
The default value is probably better (Security Support), since that what most users are looking for. "Supported" can be confusing.

```suggestion
```

adriens committed:
Update products/fairphone.md

Co-authored-by: Nemo <commits@captnemo.in>

adriens committed:
fairphone 1 : better dates

According to https://en.wikipedia.org/wiki/Fairphone_1

> The last batch of devices of the second production run was sold in February 2015. No more Fairphone 1 were produced, with the company focusing on developing the phone's successor Fairphone 2

> In July 2017, Fairphone announced in an e-mail to their buyers that they were sorry not to be able to pursue the release of Android 4.4 “KitKat” for the Fairphone 1

adriens committed:
Better data for In July 2017, Fairphone announced in an e-mail to their buyers that they were sorry not to be able to pursue the release of Android 4.4 “KitKat” for the Fairphone 1

As of https://en.wikipedia.org/wiki/Fairphone_2  :

>  and has since received both hardware improvements and major software updates, initially shipping with Android 5 "Lollipop" and running Android 10 "Queen Cake" as of November 2021

As of https://www.fairphone.com/en/2019/03/22/proving-our-case-fairphone-2-is-sold-out/

> , while we have one remaining batch of refurbished units which will come back in stock in the coming weeks, but when those are sold, it will mark the end of Fairphone 2 sales. While the Fairphone 2 will no longer be available in our webshop as of now, we will continue to support the Fairphone 2 and the sale of Fairphone 2 spare parts.

adriens committed:
Fairphone 3

As of https://www.fairphone.com/en/2021/09/02/fairphone-3-update/

> End of sale does not mean end of life

While we are ending the sale of the Fairphone 3, it does not mean it’s the end of its life! In fact, because of our unique approach for our Android phones, you can still upgrade your phone with camera modules and spare parts. We are also planning to update the Fairphone 3 to Android 11 in 2022

So we are in Adnroid 11 , as of https://endoflife.date/android, it is still covered by Security Support ;-p

adriens committed:
Update products/fairphone.md

Co-authored-by: Nemo <commits@captnemo.in>

adriens committed:
Update products/fairphone.md

Co-authored-by: Nemo <commits@captnemo.in>

usta commented on products/fairphone.md:
```diff
@@ -32,7 +32,6 @@ releases:
 
-> We aim to support Fairphone 4 with security and software upgrades for approximately five years from its launch
 Fairphone 4 is guaranteed an upgrade to Android 13. Updates to Android 14/15 [might happen on a best effort basis](https://support.fairphone.com/hc/en-us/articles/4405858006545-FP4-Fairphone-OS-Android-11-).
 [`ref`](https://support.fairphone.com/hc/en-us/articles/4405858006545-FP4-Fairphone-OS-Android-11-)
```
is this needed to ref again ? you did put link before for same

captn3m0 commented on products/fairphone.md:
```diff
@@ -0,0 +1,38 @@
+    discontinued: 2021-11-01
+    eol: 2019-03-22
+    releaseDate: 2015-12-01
+-   releaseCycle: "Fairphone 1"
```
Use a releaseLabel to allow for easier matching for clients.

```suggestion
releaseLabel: "Fairphone __RELEASE_CYCLE__"
releases:
-   releaseCycle: "4"
    discontinued: false
    eol: false
    releaseDate: 2021-09-30
-   releaseCycle: "3+"
    discontinued: false
    eol: 2021-09-01
    releaseDate: 2020-09-30
-   releaseCycle: "3"
    discontinued: 2021-09-01
    eol: 2021-09-02
    releaseDate: 2019-09-30
-   releaseCycle: "2"
    discontinued: 2021-11-01
    eol: 2019-03-22
    releaseDate: 2015-12-01
-   releaseCycle: "1"
```

usta replied:
just because of curiosity which one looks better 3+ or 3plus ?  because when i see 3+ i always think that 3 4 5 .,..

captn3m0 commented on products/fairphone.md:
```diff
@@ -0,0 +1,38 @@
+> Fairphone is a Dutch electronics manufacturer that designs and produces smartphones with the goal of having a lower environmental footprint and better social impact than is common in the industry.
+
+Fairphone 4 is guaranteed an upgrade to Android 13. Updates to Android 14/15 [might happen on a best effort basis](https://support.fairphone.com/hc/en-us/articles/4405858006545-FP4-Fairphone-OS-Android-11-).
+[`ref`](https://support.fairphone.com/hc/en-us/articles/4405858006545-FP4-Fairphone-OS-Android-11-)
```
```suggestion
```

adriens committed:
Update products/fairphone.md

Co-authored-by: Nemo <commits@captnemo.in>

adriens committed:
Update products/fairphone.md

Co-authored-by: Nemo <commits@captnemo.in>

adriens committed:
Better description of wjhat Fairphone is

adriens committed:
Inocorrect dates

cf https://github.com/endoflife-date/endoflife.date/pull/2339#issuecomment-1404639930

usta commented on products/fairphone.md:
```diff
@@ -0,0 +1,38 @@
+    discontinued: false
+    eol: false
+    releaseDate: 2021-09-30
+-   releaseCycle: "3+"
```
could you please add spaces between releasecycles

adriens replied:
:heavy_check_mark: Done

adriens committed:
add spaces between releasecycles

add spaces between releasecycles as requested by  https://github.com/endoflife-date/endoflife.date/pull/2339#discussion_r1089197242

adriens committed:
Fixing some dates

because of https://github.com/endoflife-date/endoflife.date/pull/2339#issuecomment-1407625009

adriens committed:
Better discontinued dates a FP links

as of https://github.com/endoflife-date/endoflife.date/pull/2339#issuecomment-1407629135

captn3m0 committed:
[fairphone] Add links to table

captn3m0 committed:
[fairphone] Add table for android versions
